src/60_Samples_%26_Templates/Housing.html

```diff
@@ -197,88 +229,85 @@ <h4>View on map</h4>
           </amp-iframe>
         </section>
         <section>
-          <h4>Floorplan</h4>
+          <h2>Floorplan</h2>
           <amp-img src="/img/floorplan_518x340.jpg" width="518" height="340"></amp-img>
         </section>
-        <section>
-          <h4>Mortgage Calculator</h4>
-          <form class="px1 pt1" method="GET" action="/samples_templates/housing/calculate-mortgage" action-xhr="/samples_templates/housing/calculate-mortgage-xhr"  target="_top">
-            <div class="flex mb1">
-              <span class="form-field-left">
-                Price
-              </span>
-              <div class="form-field-right relative">
-                <span>&#163</span><input type="number" name="price" maxlength="10" value="2000000" required>
-              </div>
-            </div>
-            <div class="flex mb1">
-              <span class="form-field-left">
-                Deposit
-              </span>
-              <div class="form-field-right relative">
-                <span>&#163</span><input type="number" name="deposit" maxlength="10" value="100000" required>
-              </div>
-            </div>
-            <div class="flex mb1">
-              <span class="form-field-left">
-                Annual interest
-              </span>
-              <div class="form-field-right relative">
-                <span>&#37;</span><input type="number" name="annual_interest" maxlength="10" value="2.5" step="0.01" required>
-              </div>
-            </div>
-            <div class="flex mb1">
-              <span class="form-field-left">
-                Repayment period (years)
-              </span>
-              <div class="form-field-right relative">
-                <span> </span><input type="number" name="repayment_period" value="25" required>
-              </div>
-            </div>
-            <div class="flex mb1">
-              <input type="submit" value="Calculate" class="ampstart-btn caps">
-            </div>
-            <div submit-success>
-              <template type="amp-mustache">
-                Monthly repayment {{monthly_repayment}}
-              </template>
-            </div>
-            <div submit-error>
-              <template type="amp-mustache">
-                Error! You entered some incorrect details {{err}}
-              </template>
-            </div>
-          </form>
-        </section>
       </amp-accordion>
     </div>
 
-    <h4 class="m1">Properties sold nearby</h4>
+    <!-- ## Mortgage calculator -->
+    <!-- Use `amp-form` to implement a mortgage calculator. Learn more about `amp-form` [here](/components/amp-form/). -->
+
+    <div>
+      <h2>Mortgage Calculator</h2>
+      <form method="GET" action="/samples_templates/housing/calculate-mortgage" action-xhr="/samples_templates/housing/calculate-mortgage-xhr"  target="_top">
+        <div>
+          <label for="price">
```

Comment:
FYI: you can also nest the input inside the label. Then you don't need the `for` attribute.

Reply:
done